From ac40783470b70ad63f67f211fc7d992bb4649c7d Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 23 Jul 2015 18:49:05 +0800
Subject: [PATCH] MLK-11281-4 ARM: imx: add iomuxc lpsr gpr node for imx7d

Add LPSR GPR node to support LPSR mode on i.MX7D.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 7e2658f..ce08b19 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -476,6 +476,11 @@
 				#interrupt-cells = <2>;
 			};
 
+			iomuxc_lpsr_gpr: lpsr-gpr@30270000 {
+				compatible = "fsl,imx7d-lpsr-gpr";
+				reg = <0x30270000 0x10000>;
+			};
+
 			wdog1: wdog@30280000 {
 				compatible = "fsl,imx7d-wdt", "fsl,imx21-wdt";
 				reg = <0x30280000 0x10000>;
-- 
1.8.0

